share/spice/syntax/syntax.handlebars

```diff
@@ -0,0 +1,3 @@
+
+{{description}}
```

Comment:
If you're using the default template then we should be able to get away without a `.handlebars` file